packages/discovery-provider/src/tasks/metadata.py

```diff
@@ -195,13 +195,13 @@ class PlaylistMetadata(TypedDict):
 
 
 playlist_metadata_format: PlaylistMetadata = {
-    "playlist_id": None,
```

Comment:
we do not want to pass in a `playlist_id` to an update call ever. `playlist_id` should be immutable - will add a test case for this in a follow-up.